.github/workflows/tests.yml

```diff
@@ -58,6 +58,7 @@ jobs:
         run: |
           cd ${GITHUB_WORKSPACE}/..
           ${SLIC_BIN} here
+          ${SLIC_BIN} php-version set 8.2 --skip-rebuild
```

Comment:
This is the PHP version that is running in slic right?  I realize that everything should work on all PHP versions but wouldn't we want to run the tests against the minimum supported version instead of the most recent?